packages/shopware-6-client/src/index.ts

```diff
@@ -1,12 +1,14 @@
 import { ClientSettings, setupConfig, updateConfig } from "./settings";
 import { reloadConfiguration } from "./apiService";
 
-export { config } from "./settings";
+export { config } from "./settings";import { Cart } from "../interfaces/models/checkout/cart/Cart";
```

Comment:
I believe this import is unnecessary here
```suggestion
export { config } from "./settings";
```